From 554c392ff077405e740dd9a3b79012d83701db82 Mon Sep 17 00:00:00 2001
From: Peter Seiderer <ps.report@gmx.net>
Date: Sun, 22 Mar 2020 11:48:31 +0100
Subject: [PATCH 2/3] pan_bo.h: add time.h include for time_t
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Fixes:

  ../src/gallium/drivers/panfrost/pan_bo.h:93:9: error: unknown type name ‘time_t’

Signed-off-by: Peter Seiderer <ps.report@gmx.net>
Reviewed-by: Alyssa Rosenzweig <alyssa.rosenzweig@collabora.com>
Part-of: <https://gitlab.freedesktop.org/mesa/mesa/-/merge_requests/4279>
---
 src/panfrost/encoder/pan_bo.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/panfrost/encoder/pan_bo.h b/src/panfrost/encoder/pan_bo.h
index fc20ceed1d4..7773b2baaab 100644
--- a/src/panfrost/encoder/pan_bo.h
+++ b/src/panfrost/encoder/pan_bo.h
@@ -29,6 +29,7 @@
 #include <panfrost-misc.h>
 #include "util/list.h"
 #include "pan_device.h"
+#include <time.h>
 
 /* Flags for allocated memory */
 
-- 
2.20.1

